site/en/tutorials/function_calling_python_quickstart.ipynb

```diff
@@ -3,18 +3,18 @@
     {
```

Comment:
<p>"automatically if" -&gt; "automatically, if" [add comma to avoid misreading]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/280/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>